database/rrd.h

```diff
@@ -999,13 +999,13 @@ extern void rrdhost_free_all(void);
 extern void rrdhost_save_all(void);
 extern void rrdhost_cleanup_all(void);
 
-extern void rrdhost_cleanup_orphan_hosts_nolock(RRDHOST *protected);
+extern void rrdhost_cleanup_orphan_hosts_nolock(RRDHOST *protected_host);
```

Comment:
why not update `void rrdhost_cleanup_orphan_hosts_nolock(RRDHOST *protected) {` in `rrdhost.c` as well?